locale/Makefile

```diff
@@ -31,32 +32,28 @@ all-mo: $(MOFILES)
 check: $(POXFILES)
 	msgfmt -c ${POTFILE}
```

Comment:
Also if you can add a `clean` target similarly to:

https://github.com/theforeman/foreman_discovery/pull/127/files